src/Test/StateMachine/Markov.hs

```diff
@@ -285,24 +288,54 @@ toMatrix m = do
 
 ------------------------------------------------------------------------
 
+instance IsString str => MonadFail (Either str) where
```

Comment:
Not used? Also not worth `{-# OPTIONS_GHC -Wno-orphans #-}`?

Reply:
see https://ghc.haskell.org/trac/ghc/ticket/12160